docs/kyma-environment-broker/03-03-hyperscaler-account-pool.md

```diff
@@ -0,0 +1,25 @@
+# Hyperscaler Account Pool
+​
+When provisioning clusters through Gardener with the Runtime Provisioner, the Kyma Environment Broker (KEB) requires a hyperscaler (GCP, Azure, AWS, etc.) account/subscription, but setting it up is outside of the Runtime Provisioner's scope. In order for the user to be able to easily provision Kyma clusters, Hyperscaler Account Pool was created. 
+
+It contains credentials for the hyperscaler accounts that have been set up in advance, stored in Kubernetes Secrets. The credentials are stored separately for each provider and each tenant. The content of the credentials Secrets may vary for different use cases. The Secrets are labeled with the **hyperscaler-type** and **tenant-name** labels to manage pools of credentials for use by the provisioning process. This way, the in-use and unassigned credentials (i.e. credentials available for use) are tracked. Only the **hyperscaler-type** label is added during Secret creation, and the **tenant-name** label is added when the account respective for a given Secret is claimed. The Hyperscaler Account Pool is unaware of the Secrets' content.
```

Comment:
The HAP stores the credentials for hyperscaler accounts that have been set up in advance as Kubernetes Secrets.